deps/rabbitmq_prometheus/src/rabbit_prometheus_handler.erl

```diff
@@ -106,14 +108,14 @@ format_metrics(Request, Registry) ->
     encode_format(ContentType, binary_to_list(Encoding), Scrape, Registry).
 
 render_format(ContentType, Registry) ->
-    TelemetryRegistry = telemetry_registry(),
+    %TelemetryRegistry = telemetry_registry(),
```

Comment:
If we don't need this registry any more, can it be deleted?

Reply:
Thanks for catching that.